CHANGELOG.md

```
@@ -1,24 +1,27 @@
 # Changelog
 
 <!--
-    Please refer to https://github.com/truefoundry/elasti/blob/main/CONTRIBUTING.md#Changelog and follow the guidelines before adding a new entry.
+    Please refer to https://github.com/truefoundry/KubeElasti/blob/main/CONTRIBUTING.md#Changelog and follow the guidelines before adding a new entry.
 -->
 
 ## 0.1.14
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)